src/main/java/com/google/codeu/servlets/UserLevelServlet.java

```diff
@@ -163,14 +163,17 @@ public String stripAnswer(String userAnswer){
         String result = "";
         if(userAnswer.indexOf(' ') != -1){
             String firstWord = userAnswer.substring(0, userAnswer.indexOf(' '));
-
+            System.out.println("First word: " + firstWord);
             //if the first word of the user answer equals an, a, or the, remove from string
             if(firstWord.equals("AN") || firstWord.equals("A") || firstWord.equals("THE")){
                 result = userAnswer.substring(userAnswer.indexOf(' ') + 1);
+                result.trim();
+                return result;
             }
         }
-        
+        result = userAnswer;
         result = result.trim();
+        System.out.println("Result: " + result);
```

Comment:
make sure you delete print statements when you finish! (this is super nit picky so feel free to ignore for now)

Reply:
oh thank you I forgot about this one. I'll take it out!